### PR TITLE
✨ feat(autoinstrumentation): prometheus expoter

### PR DESCRIPTION
**Pull request by halleystar**
promethues exporter used for python metrics

**Description:** 

**Link to tracking Issue(s):** <Issue number if applicable>

- Resolves: #3122

**Testing:** <Describe what testing was performed and which tests were added.>
autoinstrumentation python agent to my python project after compile with `opentelemetry-exporter-prometheus==0.46b0` working fine.

**Documentation:**

### CODE REVIEW

**Review by iblancasa — APPROVED** [review]

**Review by iblancasa — COMMENTED** [review]
I think you should add one changelog file

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,7 @@
 ### 💡 Enhancements 💡
 
 - `collector, target allocator, opamp`: Require Go 1.22 for building the operator (#2757)
+- `autoinstrumentation`: Python Automatic Instrumentation support for prometheus exporter (#3122)
```
Comment (pavolloffay) [inline CHANGELOG.md:10]: It does not belong here please create new changelog entry with `make chlog-new`

Comment (halleystar) [reply CHANGELOG.md]: sorry for that! I corrected it.

File: .chloggen/feat_python-prometheus-exporter.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+change_type: enhancement
+  
+# The name of the component, or a single word describing the area of concern, (e.g. collector, target allocator, auto-instrumentation, opamp, github action)
+component: auto-instrumentation
+
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Python Automatic Instrumentation support for prometheus exporter
+
+# One or more tracking issues related to the change
+issues: ["#3122"]
```
[the remark this answers is not in the document]
Comment (pavolloffay) [reply .chloggen/feat_python-prometheus-exporter.yaml:11]: the CI failed bc of that

[the remark this answers is not in the document]
Comment (halleystar) [reply .chloggen/feat_python-prometheus-exporter.yaml]: done for that

**Review by frzifus — APPROVED** [review]
Do we need some kind of e2e tests?

**Review by pavolloffay — APPROVED** [review]

**Comment by pavolloffay** [comment]
@frzifus it's a tracked tech dept

**Pull request merged**